src/devices/Nmea0183/Sentences/NmeaSentence.cs

```diff
@@ -67,6 +67,7 @@ protected NmeaSentence(TalkerId talker, SentenceId id, DateTimeOffset time)
         public TalkerId TalkerId
         {
             get;
+            set;
```

Comment:
why settable? could this be i.e. `init` or `required` property?

Reply:
Changed